src/components/Auth/ErrorWindow/ErrorWindow.tsx

```diff
@@ -0,0 +1,30 @@
+import classNames from 'classnames';
+
+type Props = {
+  loadError: boolean,
+  setLoadError: (value: boolean) => void,
+};
+
+export const ErrorWindow: React.FC<Props> = ({ loadError, setLoadError }) => {
```

Comment:
can you render an error component just if you have this error? In this way, you can create here timeout and delete it on componentWillUnmount